### ISSUE TITLE
pytz package not in py2exe release

### ISSUE THREAD

**Issue by GoogleCodeExporter** [SEBDYA/googletransitdatafeed#210]
```
What steps will reproduce the problem?
1. Run FV 1.2.4
2. As feed is processing an error-type message quickly shows  with
something like, Time zone not checked, install pytz
3. All feeds I've tested so far appear to have that message embedded
4. Most of the feeds ultimately validate successfully

What is the expected output? What do you see instead?
Not certain, perhaps Time Zone needs no validation

What version of the product are you using? On what operating system?
FV 1.2.4 Windows

Please provide any additional information below.
Not a major concern as feeds validate but I want to be certain I've
installed all necessary components
```

Original issue reported on code.google.com by `view...@gmail.com` on 14 Dec 2009 at 6:41

**Comment by GoogleCodeExporter** [comment]
```
I confirmed the exe version of 1.2.4 doesn't check timezones. Perhaps the 
machine used 
for the build had pytz installed in an egg. In addition to not doing this again 
it'd 
be nice to have setup.py detect the problem.
```

Original comment by `tom.brow...@gmail.com` on 14 Dec 2009 at 7:04
- Changed title: **pytz package not in py2exe release**
- Added labels: **App-FeedValidator**, **OpSys-Windows**, **Priority-High**, **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
I understand this has been fixed in binary release 1.2.5.
```

Original comment by `arnoegw.code@gmail.com` on 6 Jan 2010 at 12:49
- Changed state: **Fixed**
